art-blocks-engine-onboarding/art-blocks-engine-101/Embroidery-on-Art-Blocks-Engine.md

```diff
@@ -0,0 +1,149 @@
+---
+order: 450
+---
+
+# Embroidery on Art Blocks Engine
+
+In addition to providing a browser-based live view and media files generated from your project script, Art Blocks Engine provides tools to embroider your generative artwork on garments and accessories.
+
+## Requirements
+
+In order to get started, you will need:
+
+- [ ]  Testnet project shell on Art Blocks Engine
+Refer to [documentation](https://docs.artblocks.io/creator-docs/art-blocks-engine-onboarding/art-blocks-engine-101/engine-project-launch/) for setup instructions*
+- [ ]  Embroidery machine and software drivers compatible with DST or PES files
+*If you are new to embroidery, we recommend the [Brother PE800](https://www.amazon.com/dp/B07C7HNX92?tag=thesprucecrafts-onsite-prod-20&linkCode=ogi&th=1&ascsubtag=4171238%7Cne17ef55f56ef441eb2184399ae4d1dd615%7CB07C7HNX92)*
+- [ ]  JavaScript utilities for manipulating SVG markup
```

Comment:
Is this something we will provide via https://github.com/ArtBlocks/embroidery-template , or am I misunderstanding?

Reply:
Gentle ping here ser @shantanubala 🙏

Reply:
I think answer here is yes based on the convo @aaronpenne and I had with @luketshannon today – we'll want to essentially pull https://github.com/ArtBlocks/embroidery-template/blob/main/README.md in to the docs repo / into https://github.com/ArtBlocks/artblocks-starter-template to live as shared references on the _creator_ side (these docs are more oriented on the partner/API integrator side). 

cc @aaronpenne @cooper-temple @ABDruid @alecdalelio